<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWindow</class>
 <widget class="QMainWindow" name="HomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mountain Bike Trail Generator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GenerateTrail">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generate Trail</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reviewTrail">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Review Trail</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Click to go to trail search</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Click to go to trail review</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>560</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Application by Brendan Cook</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
